libraries/Zigbee/src/ep/ZigbeeColorDimmableLight.h

```diff
@@ -64,24 +64,53 @@
     },                                                                                   \
   }
 
+// Color capabilities bit flags (matching ZCL spec) - can be combined with bitwise OR
+#define ZIGBEE_COLOR_CAPABILITY_HUE_SATURATION  (1 << 0)  // Bit 0: Hue/saturation supported
+#define ZIGBEE_COLOR_CAPABILITY_ENHANCED_HUE    (1 << 1)  // Bit 1: Enhanced hue supported
+#define ZIGBEE_COLOR_CAPABILITY_COLOR_LOOP      (1 << 2)  // Bit 2: Color loop supported
+#define ZIGBEE_COLOR_CAPABILITY_X_Y            (1 << 3)  // Bit 3: X/Y supported
+#define ZIGBEE_COLOR_CAPABILITY_COLOR_TEMP     (1 << 4)  // Bit 4: Color temperature supported
+
+// Color mode enum values (matching ZCL spec)
+enum ZigbeeColorMode {
+  ZIGBEE_COLOR_MODE_HUE_SATURATION = 0x00,  // CurrentHue and CurrentSaturation
+  ZIGBEE_COLOR_MODE_CURRENT_X_Y = 0x01,     // CurrentX and CurrentY
+  ZIGBEE_COLOR_MODE_TEMPERATURE = 0x02,     // ColorTemperature
+};
+
 class ZigbeeColorDimmableLight : public ZigbeeEP {
 public:
   ZigbeeColorDimmableLight(uint8_t endpoint);
   ~ZigbeeColorDimmableLight() {}
 
+  // Must be called before starting Zigbee, by default XY are selected as color mode
+  bool setLightColorCapabilities(uint16_t capabilities);
+
+  [[deprecated("Use onLightChangeRgb() instead. This will be removed in a future major version.")]]
   void onLightChange(void (*callback)(bool, uint8_t, uint8_t, uint8_t, uint8_t)) {
     _on_light_change = callback;
   }
+  void onLightChangeRgb(void (*callback)(bool, uint8_t, uint8_t, uint8_t, uint8_t)) {
```

Comment:
this does not explain what each argument of the callback is. Name them or add comment

Reply:
Done, great idea